docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -238,12 +238,12 @@ Edit and save the template in the same way as in the site editor.
 Templates and template parts that have been created or edited in the site editor or template editing mode
 are saved to the database as custom post types. To export them as theme files, follow these steps:
 
-- In the site editor, open the **More tools and options** menu.
-- Select the **Export** option to download a zip file containing the files. Unpack the files.
-- Copy the updated `index.html` file from `theme/templates/` to your theme's `templates` folder.
-- Copy template part one and two from `theme/parts/` to your theme's `parts` folder.
-- Rename the template parts to `header.html` and `footer.html`, respectively.
-- Open `index.html` and update the template part slugs in the block markup.
+-   In the site editor, open the **More tools and options** menu.
+-   Select the **Export** option to download a zip file containing the files. Unpack the files.
+-   Copy the updated `index.html` file from `theme/templates/` to your theme's `templates` folder.
+-   Copy template part one and two from `theme/parts/` to your theme's `parts` folder.
+-   Rename the template parts to `header.html` and `footer.html`, respectively.
```

Comment:
I did a quick test of the export in 5.9 RC 1 and learnt that the renaming is no longer needed, this bug has been fixed.
```suggestion
```